sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -48,6 +50,21 @@ private ConfigurationSetting CreateSetting()
             };
         }
 
+        private ConfigurationSetting CreateSetting(string key, string value, string label)
+        {
+            return new ConfigurationSetting()
+            {
+                Key = GenerateKeyId($"{key}-"),
+                Value = value,
+                Label = label,
+            };
+        }
+
+        private ConfigurationSetting CreateSettingSpecialCharacters()
+        {
+            return CreateSetting($"key-{specialChars}", $"value-{specialChars}", $"label-{specialChars}");
```

Comment:
Not important, but `key-` prefix is added twice: `CreateSetting` also adds `key-`